src/components/BooksList.jsx

```diff
@@ -0,0 +1,12 @@
+import BookController from "./BookController";
+
+const BooksList = () => {
+  return (
+    <>
+      <div>Book List Here</div>
+      <BookController />
+    </>
+  );
+};
+

```

Comment:
this component is meant to receive a list of books and render books in a loop and pass them as prop to the Book component